src/common/net/fetch-client.ts

```diff
@@ -45,7 +45,16 @@ export class FetchHttpClient extends HttpClient implements HttpClientInterface {
       options.params,
     );
 
-    return await this.fetchRequest(resourceURL, 'GET', null, options.headers);
+    if (resourceURL.includes('fga/')) {
```

Comment:
Not a huge deal since if the retries work we could use it for other requests, but is there a safer way we can check for `fga` in the URL using some sort of `startsWith` logic? I worry a resource ID could include `fga` and take this path.